config/routes.rb

```diff
@@ -98,10 +98,10 @@
           post "/address-settings" => "pages/address_settings#update", as: :address_settings_update
           get "/name-settings" => "pages/name_settings#edit", as: :name_settings_edit
           post "/name-settings" => "pages/name_settings#update", as: :name_settings_update
-          get "/" => "pages#edit", as: :edit_page
-          patch "/" => "pages#update", as: :update_page
           get "/guidance" => "pages/guidance#edit", as: :guidance_edit
           post "/guidance" => "pages/guidance#update", as: :guidance_update
+          get "/question" => "pages/questions#edit", as: :edit_question
+          post "/question" => "pages/questions#update", as: :update_question
```

Comment:
Why the change from `patch` to `post`?

Reply:
This is to bring it inline with all the other add/edit question step views. We are still making `patch` request to `forms-api`  this is only the request between the browser and forms-admin.